lib/active_admin_import/importer.rb

```diff
@@ -0,0 +1,21 @@
+module ActiveAdminImport
+  class Importer
+
+    # REMOVE this after update of `active_admin_import` and `activeadmin` v1
+    # and remove correcponding line in `config/initializers/yeti.rb`
+    def batch_slice_columns(slice_columns)
+      use_indexes = []
+      headers.values.each_with_index do |val, index|
+        use_indexes << index if val.in?(slice_columns)
+      end
```

Comment:
@workgena  feel free to add this to master of active_admin_import why patching ?